blocksuite/affine/shared/src/adapters/attachment.ts

```diff
@@ -104,7 +104,7 @@ export class AttachmentAdapter extends BaseAdapter<Attachment> {
           name: item.name,
           size: item.size,
           type: item.type,
-          embed: false,
+          embed: item.type.startsWith('audio/'),
```

Comment:
we could introduce an auto embed mime types for attachments

Reply:
Not a pro on blocksuite, but I guess it might worth having a middleware somewhere to deal such logic other than composing the snapshot? Maybe @Saul-Mirone can share some thoughts on this.

Reply:
You can easly create a middleware to do that like other middleware here: https://github.com/toeverything/AFFiNE/tree/canary/blocksuite/affine/shared/src/adapters/middlewares